get.sh

```diff
@@ -115,7 +115,13 @@ getBinaryOpenjdk()
 		fi
 	else 
 		download_url=$CUSTOMIZED_SDK_URL
-		wget -q --no-check-certificate --header 'Cookie: allow-download=1' ${download_url}
+		if [[ "$PLATFORM" == "s390x_zos" ]];
+		then
+			curl -OLJ ${download_url} 
+			jar_file_name=`ls`
```

Comment:
Same this jar_file_name is  used to check if download is successful when redirection happens. Not used for this case please remove this line.

Reply:
When we are using a customized sdk, e.g., when running in Grinder, jar_file_name needs to be defined here, I tested without it, and it fails as it can't find the sdk.

Reply:
ah, that's because any issue, ```if [[ $jar_name == *zip || $jar_file_name == *jar ]]; then
				unzip -q $jar_file_name -d .
			```else
jar_file_name should be updated to jar_name, which is in @llxia PR.

Reply:
https://github.com/AdoptOpenJDK/openjdk-tests/pull/486

Reply:
Merged https://github.com/AdoptOpenJDK/openjdk-tests/pull/486 with this PR and removed `jar_file_name` from here.